meetings/2021-04-26.md

```diff
@@ -0,0 +1,71 @@
+# 2021-04-26 Solid Authentication
+
+https://meet.jit.si/solid-authentication
+
+## Agenda
+
+* JSON-LD Context for client identifiers
+    * Publication URL
+    * Substance of [PR](https://github.com/solid/authentication-panel/pull/165)
+
+* OIDC and HTTPSig issues
+    * [Multiple WWW-Authenticate headers](https://github.com/solid/authentication-panel/issues/163)
+    * [Sending Access control rules in the body of 401?](https://github.com/solid/authentication-panel/issues/167)
+    * [Syntax for Realm?](https://github.com/solid/authentication-panel/issues/166)
+
+* [PR on HTTPSig](https://github.com/solid/authentication-panel/pull/168)
+    * Question: [HttpSig, Solid or Signature?](https://github.com/solid/authentication-panel/issues/155)
```

Comment:
I recommend adding the same comma to the title of the actual issue (which I note orders the options as `HttpSig, Signature, or Solid?`).

```suggestion
    * Question: [HttpSig, Solid, or Signature?](https://github.com/solid/authentication-panel/issues/155)
```